MdePkg/Include/Library/DebugLib.h

```diff
@@ -85,6 +85,20 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 #define DEBUG_LINE_NUMBER  __LINE__
 #endif
 
+//
+// Source file.
+// Default is use the to compiler provided __FILE__ macro value. The __FILE__
+// mapping can be overriden by predefining DEBUG_FILE
+//
+// Defining DEBUG_FILE to a fixed value is useful when comparing builds
+// across machine or configuration with different slash or path
+// file.
+//
+#ifdef DEBUG_FILE
+#else
+#define DEBUG_FILE  __FILE__

```

Comment:
suggest to define DEBUG_FILE as __FILE_NAME__ for clang here also, so that _ASSERT definition can be simplified.